### PR TITLE
Update kbn.js - correct typo for megabits/sec

### PR DESCRIPTION
**Pull request by marknmel**
data rate for Mbits has typo - should be megabits/sec - not megabites/sec

### CODE REVIEW

File: public/app/core/utils/kbn.js [hunk 1]
```diff
@@ -595,7 +595,7 @@ function($, _) {
           {text: 'bytes/sec',   value: 'Bps'},
           {text: 'kilobites/sec', value: 'Kbits'},
```
Comment (utkarshcmu) [inline public/app/core/utils/kbn.js]: Kilobits :)

File: public/app/core/utils/kbn.js [hunk 2]
```diff
@@ -595,7 +595,7 @@ function($, _) {
...
           {text: 'kilobytes/sec',    value: 'KBs'},
-          {text: 'megabites/sec', value: 'Mbits'},
+          {text: 'megabits/sec', value: 'Mbits'},
           {text: 'megabytes/sec',    value: 'MBs'},
           {text: 'gigabytes/sec',   value: 'GBs'},
           {text: 'gigabites/sec',   value: 'Gbits'},
```
Comment (utkarshcmu) [inline public/app/core/utils/kbn.js]: Gigabits

**Comment by marknmel** [comment]
I should have had another coffee :-) - thanks!

**Comment by utkarshcmu** [comment]
:+1:

**Comment by bergquist** [comment]
@marknmel thanks for spotting this and helping out :)

**Pull request merged**